supervisor/homeassistant/websocket.py

```diff
@@ -234,6 +241,7 @@ async def async_send_command(self, message: dict[str, Any]) -> dict[str, Any]:
         except HomeAssistantWSConnectionError:
             await self._client.close()
             self._client = None
+            raise
```

Comment:
This is no longer needed

Reply:
I won't get an exception on caller side otherwise. How do I know that something went wrong?

Reply:
hmm... okey, let's keep it for the command then, but remove it from the message

Reply:
I think it would also be better to handle it there on caller side, but I am fine with the compromise.